src/content/Home/leadership_text.md

```diff
@@ -1 +1 @@
-Some text about HKN being entirely student run. Click on a leader to send them an email.
+HKN is led entirely by dedicated undergraduate and graduate students. Leaders are selected by election every semester.
```

Comment:
Add sth abt how we are volunteering to do this work too

Reply:
Why? Isn't that sorta implied by they need to get elected?

Reply:
Yeah true